config/serverless.yml

```diff
@@ -136,6 +136,7 @@ uiSettings:
 
 # Task Manager
 xpack.task_manager.allow_reading_invalid_state: false
+xpack.task_manager.request_timeouts.update_by_query: 60000
```

Comment:
Would using a string such as `1m` be better?

Reply:
With the PR's current state it's not possible, given the config property is a number

```ts
update_by_query: schema.number({ defaultValue: 1000 * 30, min: 1000 * 10, max: 1000 * 60 * 10 })
```

However switching to a `duration` instead would allow to do it, e.g.

https://github.com/elastic/kibana/blob/74210343a2436fdeac8076dd088d46972f061a04/packages/core/elasticsearch/core-elasticsearch-server-internal/src/elasticsearch_config.ts#L45

Reply:
Yes I know :)  I used number (milliseconds) because I was not sure if the request timeout header accepts duration as well (Its type is `number | string`). Therefore I didn't take risk and used `ms`

Now I've found [this doc](https://www.elastic.co/guide/en/kibana/current/settings.html#elasticsearch-requestTimeout) though.